workspaces/core/src/combobox/Combobox.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { Fragment, ReactNode, useState } from 'react';
+import { Combobox as HUICombobox, Transition } from '@headlessui/react';
```

Comment:
I think we need to replace the abbreviation HUI :-) as it is not really a polite world in Russian

Reply:
fixed